### DISCUSSION TITLE
"Transfer Fields" Pattern

### DISCUSSION THREAD

**Discussion by waldo1001** [microsoft/alguidelines#93] [BC Patterns]
A way to transfer "some" fields from one table to the other, including:
- extensibility
- naming convention (CopyFrom (with overloading))

**Comment by waldo1001** [comment]
example: 
![image](https://user-images.githubusercontent.com/12088142/152043946-c9c64d48-f775-4cb0-8662-21f3d65417dd.png)

**Comment by TheDoubleH** [comment]
I'll take a stab at this

**Comment by waldo1001** [comment]
Testability 

you can come up with testst that compare the field-tables from one and the other to see if you didn't forget to extend field - or that there are no conflicts - definitly in a world of multiple extensions
